scarb/src/ops/package.rs

```diff
@@ -160,7 +163,10 @@ fn package_one_impl(
 
     let uncompressed_size = tar(pkg_id, recipe, &mut dst, ws)?;
 
-    // TODO(mkaput): Verify.
+    if opts.verify {
+        dst.seek(SeekFrom::Start(0))?;
```

Comment:
It would be more defensive programming if we put this `seek` call within of `run_verify`.